Client/src/Components/DatasetUpload/DatasetForm/DatasetForm.tsx

```diff
@@ -44,9 +44,10 @@ export const DatasetForm = (props: IProps): any => {
     callListMaterials()
   }, [])
 
-  const handleSubmit = (values: DatasetUploadFormValues) => {
+  const handleSubmit = (values: DatasetUploadFormValues, { resetForm }) => {
     const dataset: IDatasetModel = { ...values.meta, reference: values.reference, data: values.data }
     onSubmit(dataset)
+    //resetForm({ values: '' })
```

Comment:
If this code is not required, then remove it

Reply:
it was removed in the latest push @Mehrdadmaskull